pkg/scheduler/framework/preemption/preemption_test.go

```diff
@@ -354,3 +370,91 @@ func TestSelectCandidate(t *testing.T) {
 		})
 	}
 }
+
+func TestFindCandidates(t *testing.T) {
+	tests := []struct {
+		name               string
+		allNodes           []*framework.NodeInfo
+		pod                *v1.Pod
+		node               *v1.Node
+		nodeStatuses       map[string]*framework.Status
+		expectedError      bool
+		expectedCandidates int // Expected number of candidates
```

Comment:
the name is enough clearly 
```suggestion
		expectedCandidates int 
```

Reply:
done